src/webui/frontend/src/components/RuleEditors/RawEditor.tsx

```diff
@@ -7,23 +7,46 @@ import {
   CheckCircleOutlined,
 } from '@ant-design/icons';
 
-import * as React from 'react';
-import {connect} from 'react-redux';
-import {bindActionCreators, Dispatch} from 'redux';
+  import * as React from 'react';
```

Comment:
There seems to be a lot of trailing spaces and such. Please use `prettier` VS Code extension to auto-format and fix lint issues.